datarobot_batch_scoring/reader.py

```diff
@@ -379,6 +379,8 @@ def investigate_encoding_and_dialect(dataset, sep, ui, fast=False,
     else:
         opener = open
     with opener(dataset, 'rb') as dfile:
+        line_to_sniff = dfile.readline()
```

Comment:
What about, going back to using pandas? The core reason we moved away from it, as I understand things, was the heavyweight dependencies and the challenges getting them installed. But we have a binary executable that we ship now with all deps. So maybe using pandas again makes sense?